### DISCUSSION TITLE
Adopting unmaintained/abandoned/underrepresented projects from the general FOSS community

### DISCUSSION THREAD

**Discussion by PaxJaromeMalues** [FossifyOrg/General-Discussion#225] [Ideas]
There are many many projects out there that are niche specific or actual runners, that sadly never received the deserved attention or hit walls they couldn't pass over.

Two examples that I would like to suggest for adoption (if this is ever considered) are these:

1. Pixy - An Open Source mood, emotion and sleep quality tracker with automatic reports for different time frames.

I am personally using the latest fdroid version, but the project is no longer maintained by the dev, even tho it had a considerable user base.

2. Yokkata - A flashcard app with audio support following a very effective training technique that originated from someone's master thesis and was abandoned due to time constraints.

kind regards
me

**Comment by naveensingh** [comment]
Thanks for your suggestions but as I mentioned [here](https://github.com/orgs/FossifyOrg/discussions/166#discussioncomment-9417721), there's enough to do as it is. New projects are tempting but we don't wanna implode even before all the apps are released.

**Reply by PaxJaromeMalues** [reply]
Understandable, have a nice week.
